src/main/java/se/sundsvall/teamssender/service/TeamsSenderService.java

```diff
@@ -0,0 +1,74 @@
+package se.sundsvall.teamssender.service;
+
+import com.microsoft.graph.models.*;
+import com.microsoft.graph.serviceclient.GraphServiceClient;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Objects;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import se.sundsvall.teamssender.api.model.SendTeamsMessageRequest;
+import se.sundsvall.teamssender.auth.service.TokenService;
+
+@Service
+public class TeamsSenderService {
+
+	@Value("${teams.sender}")
+	private String systemUser;
+
+	private final TokenService tokenService;
+
+	@Autowired
+	public TeamsSenderService(TokenService tokenService) {
+		this.tokenService = tokenService;
+	}
+
+	public void sendTeamsMessage(SendTeamsMessageRequest request) throws Exception {
+		GraphServiceClient graphClient = tokenService.initializeGraphServiceClient();
+
+		var createdChat = createChat(graphClient, systemUser, request.getRecipient());
```

Comment:
You are mixing local variable type inference (i.e. use of "var") with explicit type declaration.

Maybe choose one style and stick to it?